pkg/handlers/v1/filter_test.go

```diff
@@ -0,0 +1,91 @@
\":\"subnet-3d0b8c5a\",\"vpcId\":\"vpc-b290fcd5\",\"architecture\":\"x86_64\",\"blockDeviceMappings\":[{\"deviceName\":\"/dev/sda1\",\"ebs\":{\"attachTime\":\"2019-02-22T20:30:11.000Z\",\"deleteOnTermination\":true,\"status\":\"attached\",\"volumeId\":\"vol-0da7faa5400c54c4c\"}}],\"clientToken\":\"\",\"ebsOptimized\":false,\"enaSupport\":true,\"hypervisor\":\"xen\",\"iamInstanceProfile\":null,\"instanceLifecycle\":null,\"elasticGpuAssociations\":[],\"elasticInferenceAcceleratorAssociations\":[],\"networkInterfaces\":[{\"association\":{\"ipOwnerId\":\"amazon\",\"publicDnsName\":\"ec2-34-222-120-66.us-west-2.compute.amazonaws.com\",\"publicIp\":\"34.222.120.66\"},\"attachment\":{\"attachTime\":\"2019-02-22T20:30:10.000Z\",\"attachmentId\":\"eni-attach-0195433bad822bc2f\",\"deleteOnTermination\":true,\"deviceIndex\":0,\"status\":\"attached\"},\"description\":\"\",\"groups\":[{\"groupName\":\"launch-wizard-2\",\"groupId\":\"sg-05d1b37a375dcca8e\"}],\"ipv6Addresses\":[],\"macAddress\":\"02:17:59:ed:8b:0a\",\"networkInterfaceId\":\"eni-05721fa8354d07b8c\",\"ownerId\":\"515665915980\",\"privateDnsName\":\"ip-172-31-30-79.us-west-2.compute.internal\",\"privateIpAddress\":\"172.31.30.79\",\"privateIpAddresses\":[{\"association\":{\"ipOwnerId\":\"amazon\",\"publicDnsName\":\"ec2-34-222-120-66.us-west-2.compute.amazonaws.com\",\"publicIp\":\"34.222.120.66\"},\"primary\":true,\"privateDnsName\":\"ip-172-31-30-79.us-west-2.compute.internal\",\"privateIpAddress\":\"172.31.30.79\"}],\"sourceDestCheck\":true,\"status\":\"in-use\",\"subnetId\":\"subnet-3d0b8c5a\",\"vpcId\":\"vpc-b290fcd5\"}],\"rootDeviceName\":\"/dev/sda1\",\"rootDeviceType\":\"ebs\",\"securityGroups\":[{\"groupName\":\"launch-wizard-2\",\"groupId\":\"sg-05d1b37a375dcca8e\"}],\"sourceDestCheck\":true,\"spotInstanceRequestId\":null,\"sriovNetSupport\":null,\"stateReason\":null,\"tags\":[{\"key\":\"business_unit\",\"value\":\"CISO-Security\"},{\"key\":\"service_name\",\"value\":\"foo-bar\"}],\"virtualizationType\":\"hvm\",\"cpuOptions\":{\"coreCount\":1,\"threadsPerCore\":1},\"capacityReservationId\":null,\"capacityReservationSpecification\":null,\"hibernationOptions\":{\"configured\":false},\"licenses\":[]},\"supplementaryConfiguration\":{},\"tags\":{\"service_name\":\"foo-bar\",\"business_unit\":\"CISO-Security\"},\"configurationItemVersion\":\"1.3\",\"configurationItemCaptureTime\":\"2019-02-22T20:43:10.208Z\",\"configurationStateId\":1550868190208,\"awsAccountId\":\"515665915980\",\"configurationItemStatus\":\"ResourceDiscovered\",\"resourceType\":\"AWS::EC2::Instance\",\"resourceId\":\"i-0a763ac3ee37d8d2b\",\"resourceName\":null,\"ARN\":\"arn:aws:ec2:us-west-2:515665915980:instance/i-0a763ac3ee37d8d2b\",\"awsRegion\":\"us-west-2\",\"availabilityZone\":\"us-west-2a\",\"configurationStateMd5Hash\":\"\",\"resourceCreationTime\":\"2019-02-22T20:30:10.000Z\"},\"notificationCreationTime\":\"2019-02-22T20:43:11.256Z\",\"messageType\":\"ConfigurationItemChangeNotification\",\"recordVersion\":\"1.3\"}"
```

Comment:
Wondering if this might look less scary if it was at the bottom of the file or if it were in another file and read in. Also, is there any identifying information, like account IDs, IPs, etc, in this event or is it all dummy data?

Reply:
I'll move these structs to files. I'm pretty confident all this data is from a test account, but I'll replace it with dummy data just in case

Reply:
Fixed.